xml/dhcp_vendor_class.xml

```diff
@@ -367,6 +367,52 @@
     <param pos="0" name="os.vendor" value="Cisco"/>
   </fingerprint>
 
+  <fingerprint pattern="^CISCO SPA8800$">
+    <description>Cisco SPA8800 VoIP Gateway</description>
+    <example>CISCO SPA8800</example>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.product" value="SPA8800 IP Telephony Gateway"/>
+    <param pos="0" name="hw.cpe23" value="cpe:/h:cisco:spa8800_ip_telephony_gateway:-"/>
```

Comment:
It's interesting that the `/h` and the `/o` don't match for the spa8800, but they do for the spa8000. 
```
h:cisco:spa8000_8-port_ip_telephony_gateway
h:cisco:spa8800_ip_telephony_gateway
```
```
o:cisco:spa8000_8-port_ip_telephony_gateway_firmware
o:cisco:spa8800_8-port_ip_telephony_gateway_firmware
```